src/main/java/com/dddkurlyclone/product/domain/Product.java

```diff
@@ -0,0 +1,21 @@
+package com.dddkurlyclone.product.domain;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Product {
+  @Id @GeneratedValue private Long id;
```

Comment:
```suggestion
  @Id
  @GeneratedValue
  private Long id;
```

줄을 줄이고싶은 욕망이 있지만 그래도 이렇게 하는게 가독성이 좋을거라 생각됩니다 ㅎㅎ

Reply:
띠용 저게 저렇게 접혔네요